sources/packages/web/src/components/aest/students/ApplicationHeaderTitle.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <detail-header :headerMap="headerMap" />
+</template>
+
+<script lang="ts">
+import { defineComponent, onMounted, ref } from "vue";
+import { ApplicationSupplementalDataAPIOutDTO } from "@/services/http/dto";
+import { ApplicationService } from "@/services/ApplicationService";
+
+import DetailHeader from "@/components/generic/DetailHeader.vue";
+
+export default defineComponent({
+  components: { DetailHeader },
+  props: {
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
+    const applicationData = ref<ApplicationSupplementalDataAPIOutDTO>();
+
+    const mapApplicationHeader = (
+      application: ApplicationSupplementalDataAPIOutDTO,
+    ): Record<string, string> => {
+      if (
+        application.applicationStartDate &&
+        application.applicationEndDate &&
+        application.applicationOfferingIntensity
+      ) {
+        return {
+          Name: application.studentFullName,
+          "Application #": application.applicationNumber,
+          Institution: application.applicationInstitutionName,
+          "Study dates": `${application.applicationStartDate} - ${application.applicationEndDate}`,
```

Comment:
The date formats should be applied in the Vue side.

Reply:
Moved where the date is being formatted.